ansible/roles/test/files/ptftests/IP_decap_test.py

```diff
@@ -110,32 +113,33 @@ def print_summary(self):
 
         sys.stdout.flush()
 
-    def create_ipv4_inner_pkt_only(self, src_ip, dst_ip, tos, encap=False):
+    def create_ipv4_inner_pkt_only(self, src_ip, dst_ip, tos, encap=False, ttl=64):
         """Creates an IP only packet for the test
         @param src_ip: source ip
         @param dst_ip: destination ip
         @param tos: type of service field
         @param encap: build encapsulated packet.
                       If @encap is True the return packet would be:
                       IP(@src_ip, @dst_ip, @tos) / IP(dst_ip=4.4.4.4, src_ip=3.3.3.3) / TCP()
+        @param ttl: ttl field
         """
-
-        inner_pkt = simple_ip_only_packet(ip_dst=dst_ip, ip_src=src_ip, ip_ttl=64, ip_tos=tos)
+            
```

Comment:
can you please remove extra spaces at the end of the line

Reply:
Removed all trailing spaces.